app/models/concerns/pxe_loader_support.rb

```diff
@@ -28,7 +28,10 @@ def all_loaders_map(precision = 'x64', httpboot_host = "httpboot_host")
         "Grub2 UEFI HTTP" => "http://#{httpboot_host}/httpboot/grub2/grub#{precision}.efi",
         "Grub2 UEFI HTTPS" => "https://#{httpboot_host}/httpboot/grub2/grub#{precision}.efi",
         "Grub2 UEFI HTTPS SecureBoot" => "https://#{httpboot_host}/httpboot/grub2/shim#{precision}.efi",
-        "iPXE UEFI HTTP" => "http://#{httpboot_host}/httpboot/ipxe-#{precision}.efi"
+        "iPXE" => nil,
+        "iPXE UEFI HTTP" => "http://#{httpboot_host}/httpboot/ipxe-#{precision}.efi",
+        "iPXE Chain BIOS" => "undionly.kpxe",
+        "iPXE Chain UEFI" => "ipxe.efi"
       }.freeze
```

Comment:
Please also modify the `PXE_KINDS` hash above to match `undionly.kpxe` and `ipxe.efi`. Add few unit tests, we need to be sure mapping is correct from both directions.